.github/workflows/buildAndTest.yml

```diff
@@ -0,0 +1,78 @@
+name: Build & Test
+on:
+  pull_request:
```

Comment:
I also don't know about this, but I suppose that this **on: pull_request** means something like "execute the following with each PR"?? Don't we wanna do it with every **push**?

Reply:
See here: https://help.github.com/en/actions/reference/events-that-trigger-workflows#pull-request-event-pull_request

The event occurs whenever a change to a PR against master happens. We could change to https://help.github.com/en/actions/reference/events-that-trigger-workflows#push-event-push.

Reply:
>pull_request
>Activity types: 
>- assigned
>- unassigned
>- labeled
>- unlabeled
>- opened
>- edited
>- closed
>- reopened
>- synchronize
>- ready_for_review
>- locked
>- unlocked 
>- review_requested 
>- review_request_removed

Seems like events we don't need to trigger a build for 😉

Reply:
Switched to push in https://github.com/moia-dev/scala-http-client/pull/52/commits/02eeeaa1f54ce082bbec1633aa8d32062f1c1612.